src/app/components/driver-list-entry/driver-list-entry.component.ts

```diff
@@ -0,0 +1,54 @@
+import { Component, OnInit, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-driver-list-entry',
+  templateUrl: './driver-list-entry.component.html',
+  styleUrls: ['./driver-list-entry.component.css']
+})
+export class DriverListEntryComponent implements OnInit {
+  
+  // README: pass in the drivers list from the parent component's html with:
+  // where [drivers] matches this component variable and "drivers" matches parent's
+  // <app-driver-list-entry [drivers]="drivers"></app-driver-list-entry>
+
+  // data is expected in following format with at least these fields included:
+  // this can be deleted once we have the parent feed set up
+  driversTest = [{
```

Comment:
Perhaps we can make a model to represent this and type the array?

Documentation > single line comments for the "README" comment

Reply:
let me reach out to @chrisrchar , I am not sure how his team will be passing the data to this child component. Based on that I can update the code.

Reply:
@carolynrehm , still have not talked to Chris, but if the filtering team ends up adding the car object to the user model (which currently is no included) we can use the user model. The example I put in the code is of the minimum information the child will need passed in order to do its task. Also, the comments can be removed once we have fleshed out the parent/ child interaction. Maybe I am misunderstanding?

Reply:
Hello @carolynrehm , so after talking to the filtering team, we decided to use the user template (that filtering team is updating). I also had to change the behavior since only one driver is passed in at the time (not the whole list). Added the #docregion to the readme comment